build.sbt

```diff
@@ -86,6 +88,19 @@ val sparkDependencies = Seq (
     force()
 )
 
+val kafkaDependencies = Seq(
+  "org.apache.kafka" %% "kafka" % kafkaVersion % "optional"
+    exclude("org.slf4j", "slf4j-log4j12")
+    exclude("log4j", "log4j")
+    force(),
+  "io.airlift" % "airline" % airlineVersion % "optional",
+  "io.druid.extensions" % "druid-histogram" % druidVersion % "optional",
+  "org.apache.logging.log4j" % "log4j-api" % "2.3" % "optional",
+  "org.apache.logging.log4j" % "log4j-core" % "2.3" % "optional",
+  "org.apache.logging.log4j" % "log4j-slf4j-impl" % "2.3" % "optional",
+  "org.apache.logging.log4j" % "log4j-1.2-api" % "2.3" % "optional"
```

Comment:
Please use logback to be consistent with the rest of tranquility (you can do `++ loggingDependencies` to get it)